docs/codeql/codeql-cli/getting-started-with-the-codeql-cli.rst

```diff
@@ -47,22 +47,11 @@ Conditions <https://securitylab.github.com/tools/codeql/license>`__.
 
 .. pull-quote:: Important
 
-   There are different versions of the CLI available to download, depending
+   There are several different versions of the CLI available to download, depending
    on your use case:
 
-   - If you want to use the most up to date CodeQL tools and features, download the
-     version tagged ``latest``.
-
-   - If you want to create CodeQL databases to upload to LGTM Enterprise, download
-     the version that is compatible with the relevant LGTM Enterprise version
-     number. Compatibility information is included in the description for each
-     release on the `CodeQL CLI releases page
-     <https://github.com/github/codeql-cli-binaries/releases>`__ on GitHub. Using the
-     correct version of the CLI ensures that your CodeQL databases are
-     compatible with your version of LGTM Enterprise. For more information,
-     see `Preparing CodeQL databases to upload to LGTM
-     <https://help.semmle.com/lgtm-enterprise/admin/help/prepare-database-upload.html>`__
-     in the LGTM admin help.
+   * If you want to use the most up to date CodeQL tools and features, download the version tagged ``latest``.
+   * If you want to generate code scanning data to upload to GitHub Enterprise server, then download the version that is compatible with the CodeQL CLI used in your CI system. For more information, see "`Installing CodeQL CLI in your CI system <https://docs.github.com/en/enterprise-server@3.6/code-security/code-scanning/using-codeql-code-scanning-with-your-existing-ci-system/installing-codeql-cli-in-your-ci-system#downloading-the-codeql-cli>`__" in the GitHub documentation.
```

Comment:
My apologies. I've just remembered that we have a version of this link that always shows the latest GHES article.

```suggestion
   * If you want to generate code scanning data to upload to GitHub Enterprise server, then download the version that is compatible with the CodeQL CLI used in your CI system. For more information, see "`Installing CodeQL CLI in your CI system <https://docs.github.com/en/enterprise-server/code-security/code-scanning/using-codeql-code-scanning-with-your-existing-ci-system/installing-codeql-cli-in-your-ci-system#downloading-the-codeql-cli>`__" in the GitHub documentation.
```